routing/index_graph.hpp

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include "routing/fseg.hpp"
+#include "routing/fseg_index.hpp"
+#include "routing/geometry.hpp"
+#include "routing/joint.hpp"
+
+#include "coding/reader.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "std/cstdint.hpp"
+
+namespace routing
+{
+class JointEdge final
+{
+public:
+  JointEdge(JointId target, double weight) : target(target), weight(weight) {}
+  JointId GetTarget() const { return target; }
+  double GetWeight() const { return weight; }
+private:
+  JointId target;
```

Comment:
These fields are in the private section, mark them as consts and rename in accordance with our style guide.